patches/node/fix_handle_boringssl_and_openssl_incompatibilities.patch

```diff
@@ -295,3 +326,29 @@ index 4486d5af2c1622c7c8f44401dc3ebb986d8e3c2e..db1769f1b3f1617ed8dbbea57b5e3241
  #include <openssl/crypto.h>
  #endif  // HAVE_OPENSSL
  
+diff --git a/src/node_options.cc b/src/node_options.cc
+index b82100b6907891063a37023e6ad21f79fffade1e..d78005d41bebff4e8729400cf08ab67d27098448 100644
+--- a/src/node_options.cc
++++ b/src/node_options.cc
+@@ -5,7 +5,7 @@
+ #include "node_binding.h"
+ #include "node_external_reference.h"
+ #include "node_internals.h"
+-#if HAVE_OPENSSL
++#if HAVE_OPENSSL && !defined(OPENSSL_IS_BORINGSSL)
+ #include "openssl/opensslv.h"
+ #endif
+ 
+diff --git a/src/node_options.h b/src/node_options.h
+index 32e68086502f3eebe2dbe232b198d663db951125..cb4b6215ffbace04ecd4ad694ae00493847a7324 100644
+--- a/src/node_options.h
++++ b/src/node_options.h
+@@ -11,7 +11,7 @@
+ #include "node_mutex.h"
+ #include "util.h"
+ 
+-#if HAVE_OPENSSL
++#if 0
```

Comment:
I noticed the other `#if`s got `&& !defined(OPENSSL_IS_BORINGSSL)` added. Should this one also have it added?

Reply:
no - that doesn't work in this context unfortunately. I tried that initially.